docs/changelog/82409.yaml

```
@@ -0,0 +1,21 @@
+pr: 82409
+issues:
+  - 81210
+  - 52728
+area: Search
+type: enhancement
+summary: Allow doc-values only search on number types
+# Note: the following release highlight covers a series of PRs that added
+# this support
+highlight:
+  notable: false
+  title: Doc-values-only search is now supported on numeric, date, keyword, ip, and boolean types
+  body: |-
+    Doc-value-only fields allow users to trade slower query performance for
+    an increased indexing rate and storage space efficiency. This is
+    achieved by disabling inverted structures such as the inverted index or
+    points, and using the columnar storage format of doc values to run
+    queries instead, while still providing the same usual great performance
+    for aggregating and sorting data. {es} {version} supports term and
+    range queries on `numeric`, `date`, `keyword`, `ip`, and `boolean`
+    doc-value-only field types.
```

@ywelsch Let me know what you think of these changes. I tried to address two minor issues:

* I wasn't sure if "date" included `date_nanos`. It does, but it wasn't clear at first.
* I'm not sure we needed the info about the internal data structures and storage formats. Those felt like implementation details to me, but let me know what you think.

```suggestion
    In 8.1, you can now run `term` and `range` queries on numeric, `date`, `date_nanos`
    `keyword`, `ip`, and `boolean` fields that only store doc values.
    
    Doc-value-only fields give an increased indexing rate and better storage
    efficiency in exchange for slower query speeds. As before, doc-value-only
    fields offer great performance for aggregating and storing data.
```

---

@jrodewig I missed this ping here, sorry. I'm not sure if this made it in. Note that I've since amended the docs here: https://github.com/elastic/elasticsearch/pull/84209
It now provides an example and linkable section.

---

Awesome. Thanks, @ywelsch!